components/molecule/textareaField/src/index.js

```diff
@@ -66,7 +68,7 @@ const MoleculeTextareaField = ({
 
   const helpTextComputed = (
     <div aria-live="polite" aria-atomic="true">
-      {computeHelpText()}
+      <p className={`${BASE_CLASS}HelpText`}>{computeHelpText()}</p>
```

Comment:
In my opinion, the class should be defined as follows:
```suggestion
      <p className={`${BASE_CLASS}-helpText`}>{computeHelpText()}</p>
```